Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Utils.cs

```diff
@@ -177,5 +178,34 @@ public static void PrintToolTitle(string productName)
 
             Console.WriteLine(sb.ToString());
         }
+
+        public static string PrettyPrintJson(string json)
+        {
+            try
+            {
+                var doc = JsonDocument.Parse(json);
+                var prettyPrintJson = System.Text.Json.JsonSerializer.Serialize(doc, new JsonSerializerOptions()
+                {
+                    WriteIndented = true
+                });
+                return prettyPrintJson;
+            }
+            catch (Exception)
+            {
+                return json;
```

Comment:
Should we do anything with the exception such as let the customer know their JSON couldn't be parsed?

Reply:
For my use case it is valid for the input to not be JSON. Basically if the Lambda function returns back JSON then pretty print it, otherwise display the response as is. I renamed the method `TryPrettyPrintJson` and added documentation.